src/pages/AboutUs/Clients.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { MDBContainer, MDBCol } from 'mdbreact';
+import client1 from './clients/client-1.png';
+import client2 from './clients/client-2.png';
+import client3 from './clients/client-3.png';
+import client4 from './clients/client-4.png';
+import client5 from './clients/client-5.png';
+import client6 from './clients/client-6.png';
+import client7 from './clients/client-7.png';
+import client8 from './clients/client-8.png';
+function Detailsclient() {
+  let clientsLogo = [
+    { client: { logo: client1, name: 'myob company', id: 1 } },
+    { client: { logo: client2, name: 'Belimo company', id: 2 } },
+    { client: { logo: client3, name: 'life groups', id: 3 } },
+    { client: { logo: client4, name: 'lily company', id: 4 } },
+    { client: { logo: client5, name: 'citrus company', id: 5 } },
+    { client: { logo: client6, name: 'trusly', id: 6 } },
+    { client: { logo: client7, name: 'oldendorff company', id: 7 } },
+    { client: { logo: client8, name: 'grabuo company', id: 8 } },
```

Comment:
I don't see the need to add the client key for each object at all if they are an array i would say do it like this it will be an array of objects as you needed and this is the right practice 
`let clientsLogo = [   { logo: client1, name: 'myob company', id: 1 },
   { logo: client2, name: 'Belimo company', id: 2  },
     { logo: client3, name: 'life groups', id: 3 },
   { logo: client4, name: 'lily company', id: 4 },
   { logo: client5, name: 'citrus company', id: 5 } ,
    { logo: client6, name: 'trusly', id: 6 } ,
   { logo: client7, name: 'oldendorff company', id: 7 } ,
    { logo: client8, name: 'grabuo company', id: 8 } ];`